### ISSUE TITLE
Examples and Demo Not working on Readme.md

### ISSUE THREAD

**Issue by sdellis** [aeschylus/IIIFBookReader#1]
There are some folks who would like to contribute to this project from at least three different institutions (Princeton, Rutgers, and University of Graz in Austria).  Can you offer anything about the state of your repository and whether it is worth using as a starting point?  Some folks have added gesture support and will be updating the BookReader to the latest version of jQuery.

**Comment by datazuul** [comment]
I would like to see a working example. I cloned repo and now try to find working IIIF-URLs to be configured in the example's main.js:

    br.IIIF({
        url: 'http://dms-data.stanford.edu/data/manifests/Walters/qm670kv1873/manifest.json',
        sequenceId : 'http://dms-data.stanford.edu/data/manifests/Walters/qm670kv1873/normal',
        maxWidth: 800
    });

Could you please provide working URLs or documentation about how to configure it? What standard versions are supported (regarding manifest.json and sequenceId...)

**Comment by datazuul** [comment]
Further investigation shows, that image-URLs not working:
    "service": {
      "@id": "https://stacks.stanford.edu/image/iiif/qm670kv1873%252FW168_000009_300",

When I call this URL in browser I get

    #<NoMethodError: undefined method `stanford_only_rights_for_file' for nil:NilClass>

Or am I doing something wrong (I open the example's index.html locally from my harddisk in Firefox...)

**Comment by aeschylus** [comment]
Nope, you're not doing anything wrong. It seems it's just a dead link. I'll update this to a working (and hopefully permanent) IIIF endpoint for testing.
